src/applications/disability-benefits/all-claims/components/ArrayField.jsx

```diff
@@ -47,6 +48,8 @@ export default class ArrayField extends React.Component {
     this.state = {
       // force edit mode for any empty service period data
       editing: this.setInitialState(),
+      // track feature flag
+      revisedDisabilitiesPage: showRevisedNewDisabilitiesPage,
```

Comment:
Do we think other VA.gov pages might benefit from the updates to ArrayField behavior? Would it be more intuitive to future :tm: programmers if we used a more generalized parameter name?